tailwind.config.ts

```
@@ -81,13 +104,36 @@ export default {
+      boxShadow: {
+        star: "0px 0px 8px 2px #c77eff",
+      },
+
+      spacing: {
+        4: "4px",
+      },
     },
   },
-  plugins: [require("tailwindcss-animate"), require("tailwind-scrollbar")],
+  plugins: [
+    require("tailwindcss-animate"),
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)